packages/integration_test/README.md

```diff
@@ -7,78 +7,91 @@ and native Android instrumentation testing.
 ## Usage
 
 Add a dependency on the `integration_test` and `flutter_test` package in the
-`dev_dependencies` section of pubspec.yaml. For plugins, do this in the
-pubspec.yaml of the example app.
+`dev_dependencies` section of `pubspec.yaml`. For plugins, do this in the
+`pubspec.yaml` of the example app.
```

Comment:
I might have a rare use case, but for plugins where "web" is the only platform, I normally (almost always) need dart:html to run tests, so all my tests are "integration", and live in the "test" directory. I guess this advice makes sense for more normal packages, and for consistency across all plugins!

Reply:
I see, would it then make sense to remove this sentence?

> For plugins, do this in the `pubspec.yaml` of the example app.

I think it's kind of implicit anyway – what we're trying to tell users is put tests in `integration_tests/` of their packages as needed – if they have sub-packages like `example/`, they can have a `example/integration_tests/` directory as well

Reply:
Nah, I think web plugins are "odd" because they normally need the browser to run (some times even unit tests), so we use the integration_test package to run normal tests. I think this is documenting the current "correct" behavior. If this changes in the future, we can update.

Reply:
Got it 🙂